AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -207,7 +207,7 @@ - (void)removeAuthToken:(nullable NSString *)authToken {
   if (history) {
     MSLogDebug([MSAppCenter logTag], @"Retrieved history state.");
   } else {
-    MSLogWarning([MSAppCenter logTag], @"Failed to retrieve history state or none was found.");
+    MSLogInfo([MSAppCenter logTag], @"Failed to retrieve history state or none was found.");
```

Comment:
This is sometimes can be a real issue, I am thinking about dissociating "a decryption" issue from "there is nothing in the storage". The first one can stay as a warning and the second one can be an info.

Reply:
We should split the log message with a proper log level.